.circleci/config.yml

```diff
@@ -101,7 +101,6 @@ jobs:
       - run: make test TEST_FLAGS="-race -tags integration -timeout 5m"
```

Comment:
With https://github.com/fluxcd/flux/pull/2940/files#diff-60fe268f71ae7f8409aff40d0a3e51d8R27 is above line still necessary?

Reply:
Good point, I've removed it.